README.md

```diff
@@ -45,7 +45,7 @@ export default {
 
 That's all.
 
-Use components in templates as you would usually do but NO `import` and `component registration` required anymore! It will import components on demand, code splitting is also possible.
+Use components in templates as you would usually do but NO `import` and `component registration` required anymore! It will import components on demand, and if you use code splitting in a parent, auto-imported components will be code-split along with their parent.
```

Comment:
```suggestion
Use components in templates as you would usually do, it will import components on demand and there is no `import` and `component registration` required anymore! If you register the parent component asynchronously (or lazy route), the auto-imported components will be code-split along with their parent.
```